PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/FollowCamera.rbxmx

```diff
@@ -151,10 +149,13 @@ local function CreateFollowCamera()
 			end
 			local newLookVector = self:RotateCamera(self:GetCameraLook(), self.RotateInput)
 			self.RotateInput = Vector2.new()
+
+			local height = self:IsInFirstPerson() and 0 or self.currentHeight
```

Comment:
I see this exact line has been added in multiple camera files, can we extract it into a function instead?